contrib/qat/private_key_providers/source/qat_private_key_provider.cc

```diff
@@ -376,14 +381,16 @@ QatPrivateKeyMethodProvider::QatPrivateKeyMethodProvider(
 
```

Comment:
We should replace the exception with return at line 378 also

Reply:
done, thanks!